library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608CueBuilder.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.exoplayer.text.eia608;
+
+import android.text.Layout;
+import android.text.SpannableStringBuilder;
+import com.google.android.exoplayer.text.Cue;
+
+public class Eia608CueBuilder {
+  private SpannableStringBuilder text;
+  private float position;
+  private float line;
+  private int rowIndex;
+
+  public Eia608CueBuilder() {
+    reset();
+  }
+
+  public Eia608CueBuilder(Eia608CueBuilder other) {
+    text = new SpannableStringBuilder(other.text);
```

Comment:
I have no idea what changed, but this line started throwing exception with the same content if other.text is null. So an extra check is probably necessary here :)